packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -915,7 +915,11 @@ export class LitContracts {
 
     const networks = activeValidatorStructs.map((item: any) => {
       let proto = 'https://';
-      if (item.port !== 443) {
+      /**
+       * ports in range of 8470 - 8479 are configured for https on cayenne network
+       * we shouold resepct https on these ports as they are using OpenSSL
+       */
+      if (item.port !== 443 || (item.port < 8480 && item.port > 8469)) {
```

Comment:
Think the ge and le operators are the other way around, and need to use || instead. Or you can just ! The whole expression

Reply:
refactored the above to
```js
if (item.port !== 443 && item.port > 8480 && item.port < 8469) {
```